package.json

```diff
@@ -117,10 +117,12 @@
     "to-px": "1.0.1",
     "topojson-client": "^3.1.0",
     "webgl-context": "^2.2.0",
-    "world-calendars": "^1.0.3"
+    "world-calendars": "github:archmoj/world-calendars#0faeec8544d289c94845a73fcbb813b9bb423599"
```

Comment:
See https://github.com/alexcjohnson/world-calendars/pull/11

Reply:
Nothing more needed on the NPM side, `world-calendars` is already in the Plotly org on NPM. And IDK about "popular repository", I don't get the impression it gets any noticeable use outside of plotly.js. I'll try to transfer it.

Reply:
Thank you. Resolving.